evm/vm/forks/byzantium/computation.py

```diff
@@ -0,0 +1,33 @@
+from cytoolz import (
+    merge,
+)
+
+from evm import precompiles
+from evm.utils.address import (
+    force_bytes_to_address,
+)
+from evm.vm.forks.frontier.computation import FRONTIER_PRECOMPILES
+from evm.vm.forks.spurious_dragon.computation import SpuriousDragonComputation
+
+from .opcodes import BYZANTIUM_OPCODES
+
+BYZANTIUM_PRECOMPILES = merge(
+    FRONTIER_PRECOMPILES,
+    {
+        force_bytes_to_address(b'\x05'): precompiles.modexp,
+        force_bytes_to_address(b'\x06'): precompiles.ecadd,
+        force_bytes_to_address(b'\x07'): precompiles.ecmul,
+        force_bytes_to_address(b'\x08'): precompiles.ecpairing,
+    },
+)
+
+
+class ByzantiumComputation(SpuriousDragonComputation):
+    def __init__(self, vm_state, message):
+        super(ByzantiumComputation, self).__init__(
+            vm_state,
+            message,
+        )
+        # Override
+        self.opcodes = BYZANTIUM_OPCODES
```

Comment:
Is there a reason you are doing this in the `__init__` rather than setting them as class properties?

Reply:
oh, sorry, `BYZANTIUM_OPCODES` was passed by `VMState` during my previous implementation and I forgot to modify this part. I'll modify it to set them to class properties.